generator.rb

```diff
@@ -0,0 +1,59 @@
+def is_vowel?(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    letter = true
+  else
+    letter = false
+  end
+  return letter
+end
+
+def run_generator
+  puts "Please enter your first word for the portmanteau. It must be at least 2 characters long."
+  word_A = gets.chomp
+  until word_A.length >= 2
+    puts "I'm sorry, this word is invalid. Please enter a word that is at least 2 characters long."
+    word_A = gets.chomp
+  end
+  word_A_chop = 0
+  word_A_last = word_A.length - 1
```

Comment:
You should use vertical whitespace (empty lines) to separate different pieces of your code, the same way you would use space to separate paragraphs in English text.

Just focusing on the section for the first word, I would probably put a blank line after lines 17, 17, and 33.